openy_af4_vue_app/src/components/filters/Locations.vue

```diff
@@ -46,13 +46,51 @@ export default {
     facets: {
       type: Array,
       required: true
+    },
+    excludeByLocation: {
+      type: Array,
+      required: true
     }
   },
   data() {
     return {
       selectedLocations: this.value
     }
   },
+  computed: {
+    filteredLocations() {
+      if (!this.excludeByLocation.length) {
+        return this.locations
+      }
+      const filteredLocations = {}
+      this.locations.forEach((locationGroup, key) => {
+        // Filter out excluded locations.
+        if (!this.excludeByLocation.length) {
```

Comment:
It is impossible this condition evaluates to TRUE here... it was already checked a few lines above ^^

Reply:
@shuklina ^^^ FYI